src/bundle.js

```diff
@@ -21,6 +25,10 @@ _define("hlsjs", function() { return hlsjs; });
 var howler = require("howler");
 _define("howler", function() { return howler; });
 
+// shaka
+var shaka = require("shaka-player");
+_define("shaka", function() { return shaka });
```

Comment:
```suggestion
_define("shaka", function() { return shaka; });
```

so that style is consistent 😄